code-reviews/README.md

```diff
@@ -77,76 +74,82 @@ Code reviews should use the below guidance and checklists to ensure positive and
     * Does the PR description make sense?
     * Do all the changes logically fit in this PR, or are there unrelated changes?
     * If necessary, are the changes made reflected in updates to the README or other docs? If the changes affect how the user builds code especially.
-
-2. User Facing Changes
+1. User Facing Changes
     * If the code involves a user-facing change, is there a GIF/photo that explains the functionality? If not, it might be key to validate the PR to ensure the change does what is expected.
-    * Ensure UI changes look good without unexpected behavior.  
-
-3. Design  
+    * Ensure UI changes look good without unexpected behavior.
+1. Design
     * Do the interactions of the various pieces of code in the PR make sense?
     * Does the code recognize and incorporate architectures and coding patterns?
 
 ## Code Quality Pass
 
 1. Complexity
-    * Are functions too complex?  
+    * Are functions too complex?
     * Should a function be broken into multiple functions?
     * If a method has greater than 3 arguments, potentially overly complex.
-    * Single responsibility principle – function or class should do one ‘thing’  
-    * Can the code be understood easily by code readers?  
-    * Does the code add functionality that isn’t needed? Is it overly complex?  
-
-2. Naming/readability
+    * Single responsibility principle – function or class should do one ‘thing’
+    * Can the code be understood easily by code readers?
+    * Does the code add functionality that isn’t needed? Is it overly complex?
+1. Naming/readability
     * Did the developer pick good names for functions, variables, etc?
-
-3. Error Handling
-    * Are errors handled gracefully and explicitly where necessary?  
-
-4. Functionality  
+1. Error Handling
+    * Are errors handled gracefully and explicitly where necessary?
+1. Functionality
     * Is there parallel programming in this PR that could cause race conditions? Carefully read through this logic.
-    * Could the code be optimized? For example: are there more calls to the database than need be?  
+    * Could the code be optimized? For example: are there more calls to the database than need be?
     * If you may not fully understand how the code could affect the system, ask for help.
```

Comment:
Nit: again a bullet that is not a question in a list of questions... can we reword?  For example could it be "Is it easy to understand what the code does?     

but then maybe it fits better in the readability section.  TBH ironically I don't fully understand what this bullet is trying to say.  I know this was not one of the changes in this PR, but thinking since we are sanitizing the readme

Reply:
Good point. Fixed.